src/WebJobs.Script.WebHost/Configuration/LinuxConsumptionLegionMetricsPublisherOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public sealed class LinuxConsumptionLegionMetricsPublisherOptions
+    {
+        internal const int DefaultMetricsPublishIntervalMS = 30 * 1000;
+
+        public LinuxConsumptionLegionMetricsPublisherOptions()
+        {
+            MetricsPublishIntervalMS = DefaultMetricsPublishIntervalMS;
+        }
+
+        public int MetricsPublishIntervalMS { get; set; }
```

Comment:
```suggestion
        public int MetricsPublishIntervalMS { get; set; } = DefaultMetricsPublishIntervalMS;
```

Can remove the ctor then

Reply:
Done